libraries/imgui-opencv-poco/msvc/msvc.vcxproj

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same for project file, I vote for keeping cmake only.

Reply:
Same as above